Tasks/NugetPublisher/task.json

```diff
@@ -119,7 +119,7 @@
         "LegacyNuGetHint": "If %s is older than NuGet 3.2, you may need to edit the build definition and check the 'Path to NuGet.exe is below version 3.2' checkbox",
         "BuildIdentityPermissionsHint": "For internal feeds, make sure the build service identity '%s' [%s] has access to the feed.",
         "NoCredProviderOnAgent": "CredentialProvider.TeamBuild.exe not found on the agent. Falling back to config-based credential flow.",
-        "NotARegularFile": "%s is not a solution file. Check the 'path to solution or packages.config' property of the NuGetInstaller task.",
+        "NotARegularFile": "%s is not a valid path. Check the 'Path/Pattern to nupkg' property of the NuGetPublisher task.",
```

Comment:
I'd try to avoid the word 'valid' if possible. This message is about encountering something that's not a 'regular' file, not about not-well-formed paths or not-well-formed packages